src/api/java/baritone/api/utils/ExampleBaritoneControl.java

```diff
@@ -368,6 +369,82 @@ public boolean isInGoal(int x, int y, int z) {
             logDirect("tunneling");
             return true;
         }
+        if(msg.startsWith("tunnelcustom"))
+        {
```

Comment:
Formatting Standard: Opening braces should be on the line of some conditional, or expression. In this case: `if`.